geometry_test.go

```diff
@@ -39,6 +39,32 @@ func TestSizeUnion(t *testing.T) {
 	assert.Equal(t, size3.Height, maxH)
 }
 
+func TestSizeMax(t *testing.T) {
+	size1 := NewSize(10, 100)
+	size2 := NewSize(100, 10)
+
+	size3 := size1.Max(size2)
+
+	maxW := Max(size1.Width, size2.Width)
+	maxH := Max(size1.Height, size2.Height)
+
+	assert.Equal(t, size3.Width, maxW)
+	assert.Equal(t, size3.Height, maxH)
```

Comment:
I would prefer a literal expectation instead of computing such a trivial thing like the maximum of 10 and 100. It would make more obvious what the test expects.
Also the expectation comes first and the actual value second.
```suggestion
	assert.Equal(t, NewSize(100, 100), size1.MaxSize(size2))
	assert.Equal(t, NewSize(100, 100), size2.MaxSize(size1))
```